scripts/drupal-update.sh

```diff
@@ -0,0 +1,164 @@
+#!/bin/bash
+
+# To setup a dev build you must run:
+# $ export BUILD_ENV=dev
+# prior to starting docker container.
+
+# Get Enviroment Variable.
+build_env=$BUILD_ENV
+
+# Set drush command for container.
+drush="docker-compose run --rm php drush"
+
+# Set build type.
+dev_setup=false
+if [[ $build_env = "dev" ]]; then
+  dev_setup=true
+fi
+
+# For testing.
+echo "BUILD_ENV: $build_env"
+echo "DEV_SETUP: $dev_setup"
+
+# Start build.
+if $dev_setup ; then
+  # Dev build.
+  echo "Building Local Development Enviroment."
+  echo ""
+
+  # Check/Add Drupal settings file.
+  if [ ! -e "sites/default/settings.local.php" ]
+  then
+    echo "Copying over default datbase settings."
+    cp -v "../scripts/settings.local.dist.php" "sites/default/settings.local.php"
```

Comment:
This did not work for me, here is the error message (this is a brand new copy / paste of Drupal 8 core):
Copying over default datbase settings.
'../scripts/settings.local.dist.php' -> 'sites/default/settings.local.php'
cp: cannot create regular file 'sites/default/settings.local.php': Permission denied